CMakeLists.txt

```diff
@@ -151,6 +152,15 @@ if ((NOT LEVELDB_INCLUDE_PATH) OR (NOT LEVELDB_LIB))
     message(FATAL_ERROR "Fail to find leveldb")
 endif()
 
+if(WITH_SNAPPY)
+    find_path(SNAPPY_INCLUDE_PATH NAMES snappy.h)
+    find_library(SNAPPY_LIB NAMES snappy)
```

Comment:
Since snappy provides cmake configuration files `SnappyConfig.cmake`, we can use `find_package` instead of this to avoid debug/release mismatched issues.

Reply:
Yes, we need to check other libraries too.

Reply:
I will change this code in: https://github.com/apache/incubator-brpc/pull/1812